barretenberg/cpp/src/barretenberg/benchmark/basics_bench/basics.bench.cpp

```diff
@@ -485,4 +499,5 @@ BENCHMARK(sequential_copy)->Unit(kMicrosecond)->DenseRange(20, 25);
 BENCHMARK(uint_multiplication)->Unit(kMicrosecond)->DenseRange(12, 27);
 BENCHMARK(uint_extended_multiplication)->Unit(kMicrosecond)->DenseRange(12, 27);
 BENCHMARK(pippenger)->Unit(kMicrosecond)->DenseRange(16, 20)->Setup(DoPippengerSetup)->Iterations(5);
+BENCHMARK(bn254fr_random)->Unit(kMicrosecond)->DenseRange(10, 16);
```

Comment:
when making a zk proof do we not need randomness equal to the circuit size? It might be useful to extend this to `20` to validate something odd doesn't happen for larger circuits

Reply:
I assume we'll do it in parallel. And right now 2^20 is very slow